### DISCUSSION TITLE
How are people monitoring their tRPC procedures?

### DISCUSSION THREAD

**Discussion by victorias** [trpc/trpc#5486] [Q&A]
Hey community,

Happy tRPC user here. Wanted to ask: how are people monitoring the performance of their tRPC procedures right now?

Thanks!
V

**Comment by jamiedavenport** [comment]
hey 👋 

this is something im currently working on @ [vrpc](https://docs.vrpc.vintagelabs.app/)

would love to know what monitoring features would be useful to you and if you found a solution already

jamie

**Comment by tetranoir** [comment]
Hey, I build [iudex.ai](https://iudex.ai). We also had a problem with most observability tools dont support TRPC very well, so I built an observability tool to solve this. You can sign up here if youre interested https://uma16qrc176.typeform.com/to/kB91n8d8. Include a message about github or trpc.

<img width="802" alt="iudex_trpc_screenshot" src="https://github.com/trpc/trpc/assets/2763712/cc8fa340-0c25-4509-92a6-4e46366c4327">
